modules/common/src/Storage/AbstractDatabaseTable.php

```diff
@@ -203,16 +207,8 @@ public function count(): int {
    */
   public function query(Query $query): array {
     $this->setTable();
-    $db_query = $this->connection->select($this->getTableName(), 't')
-      ->fields('t', $query->properties);
-
-    $this->setQueryConditions($db_query, $query);
-    $this->setQueryOrderBy($db_query, $query);
-    $this->setQueryLimitAndOffset($db_query, $query);
-
-    if ($query->count) {
-      $db_query = $db_query->countQuery();
-    }
+    $query->collection = $this->getTableName();
+    $db_query = SelectFactory::create($query, $this->connection);
```

Comment:
@dafeder Somewhere in the selectFactory the table alias isn't getting passed to the the SQL select. Lets pick this up again in a co-working session